### PR TITLE
Feature/testcases

### PR DESCRIPTION
**Pull request by samuelnoye**

### CODE REVIEW

File: LoggerFrameworkProjectmainTests/LoggerTest.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+//
+
+import XCTest
+@testable import LoggerFrameworkProjectmain
+
+final class LoggerTest: XCTestCase {
+    //create SUT
+    
+    var sut: Logger!
+    var env = Logger.Environment.self
```
Comment (FitzAfful) [inline LoggerFrameworkProjectmainTests/LoggerTest.swift]: env isnt used anywhere in your tests

File: LoggerFrameworkProjectmainTests/LoggerTest.swift [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    func testLogger(){
+        //Given
+        
+        sut.log("dgg", severity: .veryLow)
+        var logMessage = sut.logMessage
+        //When
+        var result = sut.log("dgg", severity: .veryLow)
+        //Then
+        XCTAssertEqual(result, logMessage)
+    }
```
Comment (FitzAfful) [inline LoggerFrameworkProjectmainTests/LoggerTest.swift]: The method you're testing is called log so using a test method name of testLogger doesn't show what exactly you're testing. A more appropriate name would be 'testLog'

* Please leave a line before your //Given, //When and //Then

Comment (FitzAfful) [inline LoggerFrameworkProjectmainTests/LoggerTest.swift]: It is also important to test literals. 

So 
        //Given
        sut.log("dgg", severity: .veryLow)
        
        //When
        var result = sut.log("dgg", severity: .veryLow)
        
        //Then
        XCTAssertEqual(result, "The exact string that result should be")
        XCTAssertEqual(sut.logMessage, "The exact string that result should be")

File: LoggerFrameworkProjectmain/Logger.swift [hunk 3]
```diff
@@ -44,6 +45,7 @@ class Logger {
             print(logMessage )
             break
         }
+        return logMessage
     }
     func veryHigh(_ message: String){
```
Comment (FitzAfful) [inline LoggerFrameworkProjectmain/Logger.swift]: I didnt see tests for all of these. Please share your code coverage when updating or creating a PR.

**Comment by FitzAfful** [comment]
I asked Maxwell to share his comments with you. Please do same with him. Especially about commit messages etc.

File: LoggerFrameworkProjectmainTests/LoggerTest.swift (hunk 2 above)
Comment (samilaxy) [reply LoggerFrameworkProjectmainTests/LoggerTest.swift]: Okay

File: LoggerFrameworkProjectmain/Logger.swift (hunk 3 above)
Comment (samilaxy) [reply LoggerFrameworkProjectmain/Logger.swift]: Okay, sure

**Pull request merged**